cli/pack/deb_control_dir.go

```diff
@@ -48,20 +89,17 @@ func initControlDir(destDirPath string, ctx *context.Ctx) error {
 		"Depends":      "",
 	}
 
+	var deps common.PackDependencies
 	if !ctx.Tarantool.TarantoolIsEnterprise {
-		minTarantoolVersion := ctx.Tarantool.TarantoolVersion
-		maxTarantoolVersion, err := common.GetNextMajorVersion(minTarantoolVersion)
-		if err != nil {
-			return project.InternalError("Failed to get next Tarantool major version: %s", err)
+		var err error
+		if deps, err = ctx.Pack.Deps.AddTarantool(ctx.Tarantool.TarantoolVersion); err != nil {
```

Comment:
This code is common for rpm and deb, it can be performed in `pack` module once.

BTW, why do you save this to `deps` variable?
It can be `func (deps *PackDependencies) AddTarantool` that modifies list inplace.